src/assets/native.ts

```diff
@@ -55,6 +55,15 @@ const nativeAssets: { [index: string]: Asset } = {
     color: '#000000',
     decimals: 24
   },
+  SOL: {
+    name: 'Solana',
+    chain: ChainId.Solana,
+    type: 'native',
+    code: 'SOL',
+    coinGeckoId: 'solana',
+    color: '#000000',
```

Comment:
I would guess this color should be something from the logo? Like a purple of teal?